jujugui/static/gui/src/app/components/profile/navigation/navigation.js

```diff
@@ -0,0 +1,62 @@
+/* Copyright (C) 2017 Canonical Ltd. */
+
+'use strict';
+
+class ProfileNavigation extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeSection: this.props.activeSection || 'models'
+    };
+  }
+
+  componentWillReceiveProps(nextProps) {
+    this.setState({activeSection: nextProps.activeSection});
+  }
+
+  shouldComponentUpdate(nextProps, nextState) {
+    if (this.state.activeSection !== nextProps.activeSection) {
```

Comment:
Yeah good call, I had originally planned on expanding this, but I can do that at a later date if necessary